files/common/lib/systemd/system/iscsi-name-init.service

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2021 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[Unit]
+Description=One time IQN configuration for iscsid.service
+Before=open-iscsi.service
```

Comment:
Instead of introducing a dependent service, could this have been done by adding an `ExecStartPre` to the `open-iscsi.service` unit via override?

Reply:
I tried that initially but that service doesn't run on first boot.